Clarify include matcher docs on crontab resource (#5419)
chef/inspec: docs-chef-io/content/inspec/resources/crontab.md

## Syntax

> A `crontab` resource block declares a user (which defaults to the current user, if not specified), and then the details to be tested, such as the schedule elements for each crontab entry or the exact commands themselves:

Reviewer: ```suggestion
A `crontab` resource block declares a user (which defaults to the current user) and the details to be tested, such as the schedule elements for each crontab entry or the exact commands themselves:
```

    describe crontab do
      its('commands') { should include '/some/scheduled/task.sh' }
    end

> Note that the word `include` here means "the list of commands in the crontab should include this complete entry," not "a particular command should include this substring." The `include matcher always matches a complete command invocation, including options and arguments.

Reviewer: ```suggestion
{{< note >}}

The `include` matcher in this context specifies the entire list of commands that the crontab should include and not a particular substring that should be included by a command. The `include` matcher always matches a complete command invocation, including options and arguments.

{{< /note >}}
```
